Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/WorkflowCondition.java

```diff
@@ -48,8 +48,10 @@ public WorkflowCondition(String type, String value) {
             this.type = WorkflowConditionType.SCRIPT;
         } else if (type.equalsIgnoreCase("xpath")) {
             this.type = WorkflowConditionType.XPATH;
+        } else if (type.equalsIgnoreCase("none")) {
+            this.type = WorkflowConditionType.NONE;
         } else {
-            throw new InvalidParameterException("Type should be script or XPath, but was " +  type);
+            throw new InvalidParameterException("Type should be none, script or XPath, but was " +  type);
```

Comment:
I would put the `none`-condition first like it is defined in `WorkflowConditionType` for consistency. Also, it might be the most likely case anyway. We wouldn't have to evaluate all options all the time.

Might make sense to change the if/else if block to a switch-statement now.

```
switch (type.toLowerCase()) {
    case "none": { … }
    case "script": {…}
    case "xpath": {…}
    default: {Exception…}
}
```

Reply:
Actually it's the least likely case, because it's only set if you actively select the noOption in the frontend, otherwise it's null.
But 👍  for the switch statement, I'm going to change this.

Reply:
> Actually it's the least likely case, because it's only set if you actively select the noOption in the frontend, otherwise it's null.

Ah… yeah… you're right. The difference between theory and practice.😄